src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -2322,6 +2332,18 @@ private void showCommentLog() {
       }
     }
   };
+
+  private AbstractAction m_showUnitNationAction = new AbstractAction(){
```

Comment:
We have a utility library that will make most of this code not necessary:
    `SwingAction.java` -- `public static AbstractAction of(final Runnable swingAction) {`
```
SwingAction.of( e -> {
   UnitsDrawer.setN...
   geMapPanel().resetMap();
});
```